ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/extensions/KsqlResourceContext.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.extensions;
+
+import io.confluent.ksql.rest.server.KsqlRestConfig;
+import io.confluent.ksql.util.KsqlConfig;
+
+public interface KsqlResourceContext {
+
+  KsqlConfig ksqlConfig();
```

Comment:
Since `KsqlConfig` and `KsqlRestConfig` are already available, is an additional `Context` class necessary? If not, it should be avoided.